Automattic/jetpack
# Fix ThreatDataView popover content

Fixes #
The content in the IconToolTip popover is not properly styled.

## Proposed changes:

* Add CSS to apply changes

### Other information:


## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/1471

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout this branch
* Load storybook with `cd projects/js-packages/storybook && npm run storybook:dev`
* View the Threats Data View component and hover over the red info icon
* Ensure the content is position and styled appropriately

## Screenshot
![Screen Shot 2024-10-21 at 10 08 41](https://github.com/user-attachments/assets/9ee56013-debd-4c59-8bc4-2585cb52de64)
